config_ani_example.py

```diff
@@ -0,0 +1,86 @@
+## SET YOUR SETTINGS IN A COPY OF THIS CALLED config_ani.py
```

Comment:
Just out of curiosity, is there any reason why you opted to use just a python file, and not something a bit more suited to this kind of thing?

IE JSON, YAML (used alot on the team), TOML, etc.

Reply:
Personal preference and lack and needing any more libraries or code to process the config info. I can just import in the file. It also lets me put some sanitation logic at the bottom of the file, without having to break that off. But yeah, I'm not apposed to changing it if there is a reason to, lmk what you think @KavinSatheeskumar.

Reply:
I agree with Kavin on this, Python files shouldn't be used for configs. I would opt for YAML in this case but feel free to use JSON or something else if you really want. This isn't super space-critical embedded code, so we don't really need to worry about not needing external libraries